Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM python:3.8.5-slim@sha256:9c84459051ee7e9d386b24ee5468352d52b6789a5d4a8cf6a649a8a1c6ad5636
+MAINTAINER Will Kahn-Greene <willkg@mozilla.com>
```

Comment:
I put my name here because I didn't know what else to do. With other projects I've worked on, we don't specify the maintainer.

Should we specify a maintainer? Who should it be?

Reply:
Personally I would skip it, unless it's required. This is sort of thing git history is good for. :)

Reply:
:+1: